products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationMainWindow.cpp

```diff
@@ -28,10 +28,18 @@ namespace gui
 
     void RelaxationMainWindow::setSoundsList(std::vector<db::multimedia_files::MultimediaFilesRecord> sounds)
     {
+        auto trimTittle = [](const std::string &title) {
+            std::string shortTittle = title.substr(0, maxDisplayedTitleLength);
+            if (shortTittle.back() == '\342') {
```

Comment:
I'd define properly named constant here and use that for comparison. It's now a bit unclear what does this magic number mean and why is it removed specifically.

Reply:
Done.